### PR TITLE
4 api requests fr3 dmer

### PR DESCRIPTION
**Pull request by Fr3dMer**
Crated API requests object for interacting with the api

### CODE REVIEW

File: 20230_11_06_requests.py [hunk 1]
```diff
@@ -24,3 +24,7 @@
...
...de_of_inheritance': 'MONOALLELIC, autosomal or pseudoautosomal, NOT imprinted', 'mode_of_pathogenicity': None}], 'stats': {'number_of_strs': 0, 'number_of_genes': 11, 'number_of_regions': 0}, 'types': [{'name': 'Rare Disease 100K', 'slug': 'rare-disease-100k', 'description': 'Rare Disease 100K'}, {'name': 'GMS Rare Disease Virtual', 'slug': 'gms-rare-disease-virtual', 'description': 'This is a panel for the Genomic Medicine Service for an exome/genome/panel based test that requires a virtual gene panel for rare disease in the Test Directory.'}, {'name': 'GMS Rare Disease', 'slug': 'gms-rare-disease', 'description': 'This panel type is used for GMS panels that are not virtual (i.e. could be a wet lab test)'}, {'name': 'GMS signed-off', 'slug': 'gms-signed-off', 'description': 'This panel has undergone review by a NHSE GMS disease specialist group and processes to be signed-off for use within the GMS.'}], 'status': 'public', 'hash_id': '554a0ac9bb5a167e4ccd1ec1', 'regions': [], 'version': '4.0', 'disease_group': 'Skeletal disorders', 'version_created': '2023-03-22T15:38:28.046827Z', 'disease_sub_group': 'Skeletal dysplasias', 'relevant_disorders': ['R45'], 'signed_off': '2023-03-22'}
```
Comment (Fr3dMer) [inline 20230_11_06_requests.py:30]: This should be removed

Comment (AbiHaddon) [inline 20230_11_06_requests.py:30]: this should be removed

File: 20230_11_06_requests.py
Comment (AmyGrimwood) [inline 20230_11_06_requests.py:1]: This shouldn't be in repo

File: README.md
Comment (AmyGrimwood) [inline README.md:1]: Good start on readme

**Comment by Fr3dMer** [comment]
I think this is ready

**Review by AbiHaddon — COMMENTED** [review]
Happy with changes discussed

File: src/api.py
Comment (AmyGrimwood) [inline src/api.py:1]: Happy with changes discussed in person

**Review by AmyGrimwood — APPROVED** [review]

**Review by AbiHaddon — APPROVED** [review]
Happy with changes discussed

**Pull request merged**